### PR TITLE
Remove unnecessary duplication & add try/catch...

### PR DESCRIPTION
**Pull request by photodude**
- Reference the chapter on DocBlocks rather than duplicating content.
- Add a try-catch example
- Array definitions should have a trailing comma (rather than may have)
- Global Variables only need to be referenced once in the document.
- Reword the table prefix statement to be more clear.
- Update the SQL Queries stuff

### CODE REVIEW

File: manual/en-US/coding-standards/chapters/php.md [hunk 1]
```diff
@@ -184,7 +196,7 @@ echo JRoute::_(
 
 ## Arrays
 
-Assignments (the `=>` operator) in arrays may be aligned with tabs. When splitting array definitions onto several lines, the last value may also have a trailing comma. This is valid PHP syntax and helps to keep code diffs minimal.
+Assignments (the `=>` operator) in arrays may be aligned with tabs. When splitting array definitions onto several lines, the last value should also have a trailing comma. This is valid PHP syntax and helps to keep code diffs minimal.
```
Comment (mbabker) [inline manual/en-US/coding-standards/chapters/php.md]: I know it's not related but can you change the "may be aligned" part to "with spaces"?

**Comment by photodude** [comment]
As requested, changed to reflect Array assignments maybe aligned with spaces.

File: manual/en-US/coding-standards/chapters/php.md [hunk 2]
```diff
@@ -317,35 +288,17 @@ Class definitions start on a new line and the opening and closing braces are als
...
--   `@package` (required)
--   `@subpackage` (optional)
--   `@author` (optional but only permitted in non-Joomla source files, for example, included third-party libraries like Geshi)
--   `@copyright` (optional unless different from the file Docblock)
--   `@license` (optional unless different from the file Docblock)
--   `@deprecated` (optional)
--   `@link` (optional)
--   `@see` (optional)
--   `@since` (required, being the version of the software the class was introduced)
+More details on DocBlocks CLass comments can be found in the chapter on [DocBlocks Comments](coding-standards/chapters/docblocks.md).
```
Comment (zero-24) [inline manual/en-US/coding-standards/chapters/php.md]: Is "CLass" expected here or more a typo?

**Comment by zero-24** [comment]
Sorry did not saw that this is merged.

**Comment by photodude** [comment]
@zero-24 It's a typo, (working on things too late at night and bad typing habits...)

**Pull request merged**